sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -55,14 +55,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 534071e52f84bad1dd7fb210a360414507f3b3ae
-  # Merge: 9fc82304 d164792b
+  set(UNIFIED_RUNTIME_REPO "https://github.com/martygrant/unified-runtime.git")
+  # commit ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc
+  # Merge: 62e6d2f9 5fb82924
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Nov 15 10:51:54 2023 +0000
-  #     Merge pull request #1077 from fabiomestre/fabio/combines_fixes_cuda_hip
-  #     [CUDA][HIP] Combined CTS Fixes
-  set(UNIFIED_RUNTIME_TAG 534071e52f84bad1dd7fb210a360414507f3b3ae)
+  # Date:   Wed Nov 8 13:32:46 2023 +0000
+  #     Merge pull request #1022 from 0x12CC/l0_usm_error_checking_2
+  #     [UR][L0] Propagate OOM errors from `USMAllocationMakeResident`
+  set(UNIFIED_RUNTIME_TAG martin/openclCommandBuffers)
```

Comment:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 109ed46ee55f41317d35b2a9a20fa7a2029e9e64
  # Merge: 31b654f9 23005313
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Wed Nov 22 16:04:52 2023 +0000
  #     Merge pull request #966 from martygrant/martin/openclCommandBuffers
  #     [OpenCL] Add Command Buffer extension to OpenCL adapter.
  set(UNIFIED_RUNTIME_TAG 109ed46ee55f41317d35b2a9a20fa7a2029e9e64)
```

Reply:
https://github.com/intel/llvm/pull/11820 has merged now. I'll go ahead and make these changes and push them to this branch.